### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by hadia206**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,16 @@ Other packages that are required to run the data generation scripts, `pandas_dat
...
 `python path/script.py --help` shows the usage.
 
+By default all examples and data generation scripts can be run from home directory (Bodo-examples) without any changes. Otherwise, make sure to change path of data files.
+
 For more information on data generation and examples, please see the docstring at the top of each python script.
 
-- [Monte Carlo Pi Calculation](https://github.com/Bodo-inc/Bodo-examples/blob/master/examples/pi.py)
-- [k-means](https://github.com/Bodo-inc/Bodo-examples/blob/master/examples/k-means.py)
+- [Monte Carlo Pi Calculation](examples/miscellaneous/pi.py)
+- [k-means](examples/miscellaneous/k-means.py)
```
Comment (ehsantn) [inline README.md:24]: Should we move the Numpy examples down to de-emphasize?

Comment (hadia206) [reply README.md:24]: ok. will do that.

Comment (hadia206) [reply README.md]: Done.

**Review by ehsantn — APPROVED** [review]
Looks good to me. Thanks!

**Review by ritwika314 — APPROVED** [review]

**Pull request merged**